### PR TITLE
No noarch

### PR DESCRIPTION
**Pull request by CJ-Wright**
Maybe this will fix the segfault?

### CODE REVIEW

**Comment by tacaswell** [comment]
any theory as to why noarch would be related to segfaults?

**Comment by CJ-Wright** [comment]
I'm more worried by the python constraint than the noarch, but no. I haven't been able to reproduce the error on CF (or my local) so this is a bit of a shot in the dark (although I think this recipe is one of the few to actually restrict the python version with noarch so :man_shrugging: ).

File: recipes-tag/xpdacq/meta.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ requirements:
 test:
   imports:
     - xpdacq
-    - xpdacq.xpdacq
-    - xpdacq.beamtime
+#    - xpdacq.xpdacq
+#    - xpdacq.beamtime
```
Comment (mrakitin) [inline recipes-tag/xpdacq/meta.yaml]: ha, that will definitely help since it's failing on the tests.

Comment (CJ-Wright) [reply recipes-tag/xpdacq/meta.yaml]: Exactly :)

**Comment by mrakitin** [comment]
OK, let's retry it.

**Comment by mrakitin** [comment]
For completeness - it was built successfully: https://anaconda.org/lightsource2-tag/xpdacq/0.8.2/download/linux-64/xpdacq-0.8.2-py36_0.tar.bz2.

**Pull request merged**